### PR TITLE
Update publish tests to use pytest

### LINKED ISSUE #828: Convert publish tests to pytest.
Same as with previous tests the publishing tests should also be using the pytest framework.

### PR DESCRIPTION
**Pull request by hstct**
closes #828

### CODE REVIEW

File: pulp_deb/tests/functional/values.py
Comment (hstct) [inline pulp_deb/tests/functional/values.py:1]: Not sure if `values.py` is a good name for this. Open for suggestions :)

Comment (mdellweg) [reply pulp_deb/tests/functional/values.py:1]: You think constants.py does not fit for it?

Comment (hstct) [reply pulp_deb/tests/functional/values.py:1]: Uhm, true I could put it in there. In my mind I was thinking having a distinct file for expected values. But now that you mention it they are just constants and other tests use values from `constants.py` for their assertions.

Gonna change it :)

**Review by quba42 — APPROVED** [review]
Lets merge.

**Pull request merged**